cg/models/demultiplex/flow_cell.py

```diff
@@ -86,7 +86,7 @@ def sample_sheet_path(self) -> Path:
     def set_sample_sheet_path_hk(self, hk_path: Path):
         self._sample_sheet_path_hk = hk_path
 
-    def get_sample_sheet_path_hk(self) -> Optional[Path]:
```

Comment:
This type hint was not correct since an exception is raised if the path has not been set for the sample sheet